pil_analyzer/src/pil_analyzer.rs

```diff
@@ -571,25 +591,25 @@ impl<T: FieldElement> PILAnalyzer<T> {
                 Expression::LambdaExpression(LambdaExpression { params, body })
             }
             PExpression::BinaryOperation(left, op, right) => {
-                if let Some(value) = self.evaluate_binary_operation(&left, op, &right) {
-                    Expression::Number(value)
-                } else {
-                    Expression::BinaryOperation(
-                        Box::new(self.process_expression(*left)),
-                        op,
-                        Box::new(self.process_expression(*right)),
-                    )
-                }
+                // if let Some(value) = self.evaluate_binary_operation(&left, op, &right) {
```

Comment:
That's now done by the pil optimizer, right?

Reply:
Yes